index.html

```diff
@@ -1,13 +1,13 @@
-<!DOCTYPE html>
-<html>
+<!doctype html>
+<html lang="no">
   <head>
     <meta charset="utf-8" />
-    <link rel="preconnect" href="%REACT_APP_API_HOST%" />
-    <link rel="dns-prefetch" href="%REACT_APP_API_HOST%" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="%VITE_API_HOST%" />
+    <link rel="dns-prefetch" href="%VITE_API_HOST%" />
```

Comment:
Må noe prefikses med env her? Tror dette virket tidligere, men nå krasjer det i codebuild

https://vitejs.dev/guide/env-and-mode#html-env-replacement

Reply:
Ser OK ut nå 🤷

Reply:
feiler pga manglende env i codebuild. Trodde [dette](https://github.com/BIBSYSDEV/NVA-Frontend/pull/3736/commits/6ae5453f288064f032c1dd6bb64553151228e31e) ville fikse, men akk nej :(

Reply:
Joda, måtte bare oppdatere riktig fil 🙃  [a20c154](https://github.com/BIBSYSDEV/NVA-Frontend/pull/3736/commits/a20c154c3daff66aaeb76509f64e5ab90d81714f)